apps/angular-example-counter/src/app/types.ts

```diff
@@ -1,7 +1,8 @@
 import { Subject, Observable } from 'rxjs';
+import { RxStoreValue } from '@rx-store/rx-store';
 
 /** Our context value, which contains subjects & observables */
-export interface RootAppStore {
+export interface RootAppStore extends RxStoreValue {
```

Comment:
@kylecannon can you help me move the "local" state out of the store in this Angular example like I did in React? It can come from wherever, but the store is just subjects now.

https://github.com/rx-store/rx-store/pull/46/files#diff-d1d154a283b2db053d0a48dd0796dd0aR11